source/api/image/3.0/index.md

```diff
@@ -395,7 +395,9 @@ The `default` quality exists to support [level 0 compliant implementations][imag
 
 Regardless of level of compliance, services that make additional qualities beyond `default` available _MUST_ list those qualities in the [`extraQualities` property][image3-extra-functionality] of the response to the [Image Information Request][image3-information]. A request for an image with an unsupported quality value _SHOULD_ return a 400 (Bad Request) status code.
 
-**N.B.** A request for the `color` quality of an image may return an image comprised solely of grayscale or bitonal pixels if this is all of the color information the server has available. The server _SHOULD NOT_ include `color` in the `extraQualities` list when this is a case, but a request of this nature _MUST NOT_ result in an error.
+__Qualities and Color Information__<br />
+A request for the `color` quality of an image may return an image comprised solely of grayscale or bitonal pixels if this is all of the color information the server has available. The server _SHOULD NOT_ include `color` in the `extraQualities` list in such cases, but a request for the `color` quality _MUST NOT_ result in an error. Similarly, a request for the grayscale quality may return an image comprised solely of bitonal pixels.
+{: .note}
 
```

Comment:
We agree that these are not RFC MAYs, correct? i.e.:

> the `color` quality of an image may return an image comprised solely of grayscale or bitonal pixels
...
> a request for the grayscale quality may return [...] bitonal pixels

If RFC `MAY` is a synonym for OPTIONAL, then using an RFC MAY could imply that the server does not have to return an image, which would contradict:

> a request for the `color` quality _MUST NOT_ result in an error

Reply:
I agree that they are not RFC MAYs.

Is the first, though, an RFC MUST? To avoid the error the next bit says MUST NOT happen?

... MUST return an image comprised solely of grayscale or bitonal pixels **if** this is all of the color information the server has available. 

(i.e., `color` doesn't mean colorize it :) )

Reply:
How about something like:

> the `color` quality MUST return an image, although the image need not contain color pixels.

Reply:
or to combine with @tomcrane's comment:

| the `color` quality MUST return an image, although the image need not contain color pixels if no color information is available.

Reply:
>(i.e., color doesn't mean colorize it :) )

Unless it does? I have no idea what that would mean, but I think I'm more comfortable with the fuzzy semantics.

Pushing up another change...

Reply:
>I have no idea what that would mean
https://arxiv.org/abs/1406.2661